XCUITests/ToolbarTest.swift

```diff
@@ -112,8 +112,9 @@ class ToolbarTests: BaseTestCase {
             waitForExistence(app.otherElements["Navigation Toolbar"])
         }
         navigator.openURL(website1["url"]!, waitForLoading: true)
+        // Adding the waiter right after navigating to the webpage in order to make the test more stable
+        waitForExistence(app.buttons["TabLocationView.pageOptionsButton"])
```

Comment:
My logic for adding the waiter here it that before the page was loaded the app.buttons["TabLocationView.pageOptionsButton"] was added to the pageActionMenuButton variable and when it hit the waitForExistence(pageActionMenuButton, timeout: 5) it might have had a different value and cause the test to fail because the element was not found. Not sure if this makes sens but for me the tests do not fail locally.

Reply:
@SimonBasca This test is still failing for me locally, same error as in BB...It looks like that after swiping up the app and tapping on the `hiddenStatusbarElement`the toolbar menu is not shown.